lib/styles/core.less

```diff
@@ -21,8 +21,11 @@
   font-weight: 400;
   line-height: @baseLineHeight;
 
-  a:focus {
-  	outline: 3px solid @focusColor;
+  a:focus,
+  button:focus,
+  input:focus {
+    outline: 3px solid @focusColor;
```

Comment:
Should we be using ems instead?

Reply:
Probably but there's a bunch of other stuff in TopHat that's already px not ems (or rems) so followed that. We'll be migrating to the new menu in a couple of weeks anyway....